conda/pytorch-cuda/meta.yaml

```diff
@@ -1,11 +1,17 @@
-# Package to manage cuda version in PyTorch
+# Package to manage cuda version in PyTorch.
+#
+# Windows anaconda packages are packaged differently,
+# All dlls are kept within *-dev packages hence we need
+# include the dev packages for Windows see:
+# https://github.com/pytorch/vision/issues/7185#issuecomment-1420002413
+#
 # Please note: Build number should be advanced with
 # every deployment. After the deployment to production
 # use following links to validate the correctness of
 # deployment:
 # https://conda.anaconda.org/pytorch/noarch/
 # https://conda.anaconda.org/pytorch/noarch/repodata.json
-{% set build = 3 %}
+{% set build = 2 %}
```

Comment:
Looks like we are just moving faster then light, so time is spinning backward :D

Reply:
Thanks for review, we probably want to keep v2. Since in prod current latest one is v1:
https://anaconda.org/pytorch/pytorch-cuda/files

For nightlies and test I think we can keep v2.